CLion setup: Reference clang-format alternative (#4492)
godotengine/godot-docs: community/contributing/code_style_guidelines.rst

- Visual Studio Code: `Clang-Format <https://marketplace.visualstudio.com/items?itemName=xaver.clang-format>`__
- Visual Studio: `ClangFormat <https://marketplace.visualstudio.com/items?itemName=LLVMExtensions.ClangFormat>`__
- vim: `vim-clang-format <https://github.com/rhysd/vim-clang-format>`__

> - CLion: starting from version ``2019.1`` plugin is not required, just enable

Reviewer: ```suggestion
- CLion: Starting from version ``2019.1``, no plugin is required. Instead, enable
```
